docs/src/modules/components/ApiPage.js

```diff
@@ -284,6 +284,8 @@ import { ${componentName} } from '${source}';`}
           </React.Fragment>
         )}
         <Heading hash="props" />
+        <span dangerouslySetInnerHTML={{ __html: spreadHint }} />
+        <br />
```

Comment:
This one should do it. It would make it looks consistent:

<img width="548" alt="Capture d’écran 2021-06-18 à 00 57 08" src="https://user-images.githubusercontent.com/3165635/122482091-34263b00-cfd0-11eb-8de3-490e107a5af5.png">

```suggestion
        <p dangerouslySetInnerHTML={{ __html: spreadHint }} />
```

For instance, with https://deploy-preview-26778--material-ui.netlify.app/system/properties/#properties-reference-table